web/js/chatRoom.jsx

```diff
@@ -180,21 +184,21 @@ var ChatRoom = React.createClass({
           />
           <input type='submit' className='btn btn-primary' value='Get Chat' />
         </form>
-  
-        <Loader error={this.state.error} loading={this.state.loading}> 
+
+        <Loader loading={this.state.loading}>
           <div className='container-fluid h-100'>
-            <ChatRoomResults results={this.state.results}/>
+            <ChatRoomResults results={this.state.results} error={this.state.error} />
           </div>
```

Comment:
This is not a proper fix because this hides all errors.

The proper fix is to make sure the endpoint that returns the chat room in `server/users/helper.js` function `chatRooms` do not return an error when the chatroom is not found, but an empty room.

Reply:
The below is updated now.